src/main/java/org/terasology/metalrenegades/interaction/component/WaterCupComponent.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2020 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
```

Comment:
We are using a shorter license header format now (should be automatically used with 2020.x IntelliJ when using the pure Gradle setup). 

```suggestion
// Copyright 2020 The Terasology Foundation
// SPDX-License-Identifier: Apache-2.0
```

Reply:
I applied the headings change to this PRs files. I can't seem to get the new header to automatically apply to new files though, and I don't know where I would look to find this option....